deploy/olm-catalog/akka-cluster-operator/0.2.3/akka-cluster-operator.v0.2.3.clusterserviceversion.yaml

```diff
@@ -0,0 +1,336 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: akka-cluster-operator.v0.2.3
+  namespace: placeholder
+  annotations:
+    capabilities: Seamless Upgrades
+    categories: Application Runtime
+    certified: 'false'
+    containerImage: 'lightbend-docker-registry.bintray.io/lightbend/akkacluster-operator:v0.2.2'
+    createdAt: '2019-06-28T15:23:00Z'
+    description: Run Akka Cluster applications on Kubernetes.
+    repository: https://github.com/lightbend/akka-cluster-operator
+    support: 'Lightbend, Inc.'
+    alm-examples: |-
+      [
+        {
+          "apiVersion": "app.lightbend.com/v1alpha1",
+          "kind": "AkkaCluster",
+          "metadata": {
+            "name": "akka-cluster-demo"
+          },
+          "spec": {
+            "replicas": 3,
+            "template": {
+              "spec": {
+                "containers": [
+                  {
+                    "name": "main",
+                    "image": "lightbend-docker-registry.bintray.io/lightbend/akka-cluster-demo:1.1.0",
```

Comment:
Only an example but worth updating the version here?

Reply:
This version changed from `1.0.2` to `1.1.0` with [#4](https://github.com/lightbend/akka-java-cluster-openshift/pull/4) of the demo.